### PR TITLE
reimplement rgbas

### PR DESCRIPTION
**Pull request by knaaptime**
- fixes a bug when the 'scheme' argument is passed
- reimplement using matplotlib's vectorized method; much faster
- give back numpy instead of lists

### CODE REVIEW

File: mapclassify/util.py [hunk 1]
```diff
@@ -1,9 +1,11 @@
+import numpy as np
+
 from ._classify_API import classify as _classify
 
 
 def get_rgba(
     values,
-    classifier="quantiles",
+    scheme="quantiles",
```
Comment (jGaboardi) [inline mapclassify/util.py:8]: Was the `classifier` keyword part of any previous release? Or was that something new in `main` since last release?

Comment (knaaptime) [reply mapclassify/util.py:8]: the whole function is new as of like a week ago

Comment (jGaboardi) [reply mapclassify/util.py:8]: ah yeah. whoops. LOL

File: mapclassify/util.py [hunk 2]
```diff
@@ -29,12 +31,12 @@ def get_rgba(
     Returns
     -------
     numpy.array
-        array of lists with each list containing four values that define a color using
-        RGBA specification.
+        numpy array (n,4) of lists with each list containing four values that define a
```
Comment (jGaboardi) [inline mapclassify/util.py:34]:
```suggestion
        numpy array :math:`(n,4)` of lists with each list containing four values that define a
```

File: mapclassify/util.py [hunk 3]
```diff
@@ -15,7 +17,7 @@ def get_rgba(
     ----------
     values : list-like
         array of input values
-    classifier : str, optional
+    scheme : str, optional
         string description of a mapclassify classifier, by default "quantiles"
```
Comment (jGaboardi) [inline mapclassify/util.py:21]:
```suggestion
        string description of a mapclassify classifier, by default `"quantiles"`
```

**Review by jGaboardi — APPROVED** [review]

**Comment by knaaptime** [comment]
updated and renamed with an example if @martinfleis or @sjsrey want to have a look

**Review by martinfleis — APPROVED** [review]

**Pull request merged**